README.md

```diff
@@ -204,12 +204,18 @@ A breakdown of the options and methods available to the SDK.
   ```javascript
   steps: [
     {
-      type:'welcome',
-      options:{
-        title:'Open your new bank account'
+      type: 'welcome',
+      options: {
+        title: 'Open your new bank account'
+      }
+    },
+    {
+      type: 'document',
+      options: {
+        useWebcam: true // This is in beta!
```

Comment:
No good reason other than to have it documented somewhere. Do you want to remove it?